with-legend-state-supabase/app.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
No extraneous app.json, assets, babel.config.json, package.json fields, etc. per the CONTRIBUTING guidelines https://github.com/expo/examples/blob/master/contributing.md#rules